app/views/wpcc/your_details/new.html.erb

```diff
@@ -42,10 +39,7 @@
         <div class="details__field">
           <div class="details__field-label">
             <%= f.label :gender, t('wpcc.details.gender.label'), class: 'form__label-heading' %>
-            <button type="button" class="popup-tip__button" data-dough-popup-trigger>
-              <span aria-hidden="true">i</span>
-              <span class="visually-hidden"><%= t('wpcc.details.tooltip_show') %></span>
-            </button>
+            <%= render 'wpcc/shared/popup_tip_trigger' %>
```

Comment:
@tomas-stefano I have updated the file structure as you suggested for adding the tooltip partials. Can you have another look? Cheers!